packages/components/src/popover/index.js

```diff
@@ -260,11 +260,11 @@ const Popover = ( {
 	const anchorRefFallback = useRef( null );
 	const contentRef = useRef( null );
 	const containerRef = useRef();
-	const contentRect = useRef();
 	const isMobileViewport = useViewportMatch( 'medium', '<' );
 	const [ animateOrigin, setAnimateOrigin ] = useState();
 	const slot = useSlot( __unstableSlotName );
 	const isExpanded = expandOnMobile && isMobileViewport;
+	const [ containerResizeListener, contentSize ] = useResizeObserver();
```

Comment:
I guess we can't use https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver?

Reply:
Not for the moment (IE11) but this hook is widely used in Gutenberg already and it works pretty well.

Reply:
We also already have an interval running, so we could use `ResizeObserver` and have the interval as a fallback.

Reply:
Why such complexity while the hook is already there and works properly? I'd rather find a way to remove the interval personally.

Reply:
Yes, would be great if we could remove the interval